docs/components/camera/_index.md

```diff
@@ -136,7 +132,7 @@ frame = await my_camera.get_image(mime_type="image/jpeg")
 
 <br>
 
-If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`RawImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
+If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`ViamImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
```

Comment:
If it matters, `ViamImage` is still defined in `viam/media/video`. It is also imported by `camera` as well, so also not completely wrong? Not sure if there's a preference!